dcs/coalition.py

```diff
@@ -157,11 +159,19 @@ def load_from_dict(self, mission, d) -> List[StatusMessage]:
                     # units
                     for imp_unit_idx in pgroup["units"]:
                         imp_unit = pgroup["units"][imp_unit_idx]
+
+                        if imp_unit["type"] not in planes.plane_map:
+                            print("WARN:", "Unknown plane type", file=sys.stderr)
+                            plane_type = type(imp_unit["type"], (PlaneType,), {})
```

Comment:
I'm pretty wary of this approach:

```
>>> with open(os.devnull, "w") as dev_null:
...     pickle.dump(type("Bar", (Foo,), {})(), dev_null)
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
_pickle.PicklingError: Can't pickle <class '__main__.Bar'>: attribute lookup Bar on __main__ failed
```

I don't think it's important that a distinct type is created for each aircraft type, so `PlaneType()` is probably plenty good.

This also does call `type()` for every instance of the plane that isn't recognized. Experimentally that's not a problem in the repl (`id(type("Foo")) == id(type("Foo"))`, but I can't find anything in the docs of `type` that specify the behavior one way or the other so I'd prefer to not rely on it without a good reason.